samples/login-with-auth-code-demo/src/main/res/xml/network_security_config.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<network-security-config>
+    <debug-overrides>
+        <trust-anchors>
+            <!-- Trust user added CAs while debuggable only -->
+            <certificates src="user" />
+            <certificates src="system" />
+        </trust-anchors>
+    </debug-overrides>
+
+    <base-config cleartextTrafficPermitted="true">
```

Comment:
this should be set to false?

Reply:
It should be set to true to monitor network traffic with tools like proxyman, charles etc.
We are enabling it for sample app and only debug builds so it's safe